document-warehouse/test/delete-document-schema.test.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {describe, it} = require('mocha');
+const {assert} = require('chai');
+const sinon = require('sinon');
+
+const text = 'Document Schema Deleted';
+
+const {deleteDocumentSchema} = require('../delete-document-schema');
+
+// TODO(developer): Fill in the variables below before running this test.
+const projectId = 'YOUR_PROJECT_ID';
```

Comment:
If you are planning on using a real project - do you mind leveraging the `DocumentSchemaServiceClient` to infer the project w/ `getProjectId()`. 
This should be picked up with the current GCP project we have this hooked up to.

Here's an example from another sample: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/iam/deny/test/deny.test.js#L32

Ref: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/actions/runs/5859762543/job/15886428603?pr=3423#step:11:39

Reply:
Included projectId pulling for tests. Note that I added create document schema for test completeness as well.